docs/getting-started/03-cli.md

```diff
@@ -84,24 +86,25 @@ curl -v http://127.0.0.1:4010/pets/123?__dynamic=false
 
 This command creates an HTTP server that will proxy all the requests to the specified upstream server. Prism will analyze the request coming in and the response coming back from the upstream server and report the discrepancies with what's declared in the provided OpenAPI document.
 
-**Note:** Currently the proxy command _expects_ a JSON Payload for both the request and the response. If the response's `content-type` header does not indicate a JSON-ish payload (`application/json`, `application/*+json`), then the payload will be just returned as text and send back to the original client.
+Learn more about the ideas here in our [Validation Proxy guide](../guides/03-validation-proxy.md), or see below for the quick n dirty how to CLI.
 
-```
-prism proxy examples/petstore.oas2.yaml https://petstore.swagger.io/v2
+```bash
+$ prism proxy examples/petstore.oas2.yaml https://petstore.swagger.io/v2
```

Comment:
Adding the dollar sign will be a problem for the users interested in copying and pasting these command directly in their terminals. I suggest we remote it.

```suggestion
prism proxy examples/petstore.oas2.yaml https://petstore.swagger.io/v2
```

Reply:
Common convention is to use the dollar sign to distinguish commands from output when both are mixed into a single code fence. The alternative is breaking them into two code fences, which is a bit less common back can look nice.